Blockzilla/PhotonActionSheet.swift

```diff
@@ -239,6 +240,12 @@ class PhotonActionSheet: UIViewController, UITableViewDelegate, UITableViewDataS
         self.dismiss(animated: true) { callback() }
     }
     
+    @objc func didToggle(enabled: Bool) {
+        delegate?.photonActionSheetDidToggleProtection(enabled: enabled)
+        dismiss(nil)
```

Comment:
Calling dismiss with nil feels unintuitive to me, are we able to set gestureRecognizer = nil by default or does that mess up the selector syntax?